src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/DocumentGenerator.java

```diff
@@ -63,4 +63,15 @@ public void generateForRepresentedClaim(RepresentedClaimIssuedEvent event) {
 
         publisher.publishEvent(new DocumentGeneratedEvent(event.getClaim(), event.getAuthorisation(), sealedClaim));
     }
+
+    @EventListener
+    public void generateForNonRepresentedRPA(CitizenClaimIssuedEvent event) {
```

Comment:
How about `generateForCitizenRPA`? We might add representative for citizen as well in future.